db/sqldb/initiative.go

```diff
@@ -114,11 +114,25 @@ func (d *DB) UpdateInitiative(tx db.Tx, id pacta.InitiativeID, mutations ...db.U
 	return nil
 }
 
-func (d *DB) DeleteInitiative(tx db.Tx, id pacta.InitiativeID) error {
+func (d *DB) DeleteInitiative(tx db.Tx, id pacta.InitiativeID) ([]pacta.BlobURI, error) {
+	buris := []pacta.BlobURI{}
 	err := d.RunOrContinueTransaction(tx, func(db.Tx) error {
-		// TODO(grady) add owner deletions here - where the initiative is the owner of the asset/blob
-		// TODO(grady) do snapshot deletions here
-		err := d.exec(tx, `DELETE FROM initiative_invitation WHERE initiative_id = $1;`, id)
+		owner, err := d.GetOrCreateOwnerForInitiative(tx, id)
```

Comment:
Seems odd that we might create an `owner` in the course of deleting something, is there not a `GetOwner...` endpoint we can use instead?

Reply:
Ahhh I should do the same migration I did in #76, but for initiatives. I'll do that next.